.github/workflows/conventional-pr.yml

```diff
@@ -6,6 +6,8 @@ on:
     branches:
     - master
     - develop
+    - release/*
+    - release/**
```

Comment:
You don't need both of these, ** is inclusive of *. Either one is probably fine for our needs, but ** is the "literally everything" one...

```
*: Matches zero or more characters, but does not match the / character. For example, Octo* matches Octocat.
**: Matches zero or more of any character.
```

Reply:
I'll keep `release/**` and remove the other one :)

Reply:
https://internaldocs.hq.unity3d.com/yamato/yaml/triggers/#all-pull-requests-that-have-the-master-branch-as-the-target

Reply:
this one is for GitHub Actions Workflow, not for Yamato
so the reference is: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
Sorry, pasted into the wrong PR, thx